packages/react/src/primitives/types/highlightMatch.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { ViewProps } from './view';
+
+export interface HighlightMatchProps extends ViewProps {
+  /**
+   * @description
+   * The label you would like to have match highlighting
+   */
+  children: string;
+
+  /**
+   * @description
+   * A query string used to match against the label
+   */
+  query: string;
+
+  /**
+   * @description
+   * Used to define the styles for the highlighted part
+   */
+  highlightStyles?: React.CSSProperties;
```

Comment:
I don't think we've done this in other primitives right? In the nextrelease branch added `inputStyles` to *Field components, but that prop was `BaseStyles` type that was spread on the input element. Thoughts?

Reply:
You are right. We can remove it and add a classname instead. :)